usr/share/doc/find-the-command/ftc.fish

```diff
@@ -50,8 +50,11 @@ function _cnf_asroot
     if test (id -u) -ne 0
         if $_cnf_force_su
             su -c "$argv"
-        else
+        else if command -v sudo > /dev/null 2>&1 && sudo -l > /dev/null 2>&1
```

Comment:
Why do you need `sudo -l`? Seems unnecessary as that return value should always be 0

Reply:
> Why do you need `sudo -l`? Seems unnecessary as that return value should always be 0

I checked with this code

```
if sudo -l 
then echo "sudo"
else echo "not sudo"
fi
```
And when the user does not have access to sudo the code says not sudo, and when the user has access to sudo it says sudo, as to me this check is important, because the user may have sudo installed but not configured.